apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -14,13 +14,13 @@ import {
   SlideOverHeader,
   SlideOverContent,
   SlideOver,
-} from 'ui';
+} from '@colonial-collections/ui';
 import useCurrentPublisher from './useCurrentPublisher';
 import {env} from 'node:process';
 import {formatDateCreated} from './format-date-created';
 import ObjectListsMenu from './object-lists-menu';
 import {SignedIn} from '@clerk/nextjs';
-import {Notifications} from 'ui';
+import {Notifications} from '@colonial-collections/ui';
```

Comment:
There are several imports from `@colonial-collections/ui` here. Would it be possible to use one import instead, to make visually clear which functions come from this package? If not, place the imports from the same package together?